android/build.gradle

```diff
@@ -131,7 +134,7 @@ android {
     packagingOptions {
         println "Native libs debug enabled: ${debugNativeLibraries}"
         doNotStrip debugNativeLibraries ? "**/**/*.so" : ''
-        excludes = ["**/libc++_shared.so"]
+        excludes = REACT_VERSION < 65 ? ["**/libc++_shared.so"] : ["**/libc++_shared.so", "**/libfbjni.so"]
```

Comment:
It would be nice to also automatically exclude 'libfbjni.so' from app. Can you check if we can add SO to the exclude list of another module?